models/Document/Editable/Date.php

```diff
@@ -27,7 +28,7 @@ class Date extends Model\Document\Editable implements EditmodeDataInterface
      *
      * @internal
      *
-     * @var \Carbon\Carbon|null
+     * @var Carbon|null
```

Comment:
if the $date property can be \DateTimeInterface and not Carbon it should be changed here
```suggestion
     * @var \DateTimeInterface|null
```

Reply:
And also for `getData()` and `getDate()`

Reply:
No, this should stay with Carbon.